backend/src/admin/admin.controller.ts

```diff
@@ -10,7 +12,7 @@ export class AdminController {
   ) { }
 
   @UsePipes(new ValidationPipe())
-  @Post('register')
+  @Post('register/admin')
```

Comment:
should the be a "is this admin" check?

Reply:
Awaiting a response as to how admins will be added. Will add ToDo.